src/frontend/planner_test/tests/testdata/output/ch_benchmark.yaml

```diff
@@ -2170,7 +2170,7 @@
                     └─BatchFilter { predicate: (orders.o_carrier_id > 8:Int32) }
                       └─BatchScan { table: orders, columns: [orders.o_id, orders.o_d_id, orders.o_w_id, orders.o_c_id, orders.o_carrier_id], distribution: UpstreamHashShard(orders.o_w_id, orders.o_d_id, orders.o_id) }
   stream_plan: |-
-    StreamMaterialize { columns: [c_count, custdist], stream_key: [c_count], pk_columns: [custdist, c_count], pk_conflict: NoCheck }
+    StreamMaterialize { columns: [c_count, custdist], stream_key: [c_count], pk_columns: [c_count], pk_conflict: NoCheck }
```

Comment:
This testcase modification doesn't make sense. `group by c_count norder by custdist desc, c_count desc`, we couldn't remove `custdist`, even if we know that `c_count` determines `custdist`.

Reply:
Hmm yes, it should only remove suffix. Let me check it.

Reply:
Fixed